website/docs/regions_and_zones.html.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: "scaleway"
+page_title: "Scaleway Zones and Regions"
+description: |-
+  Scaleway resources can be created in availability zones and regions.
+---
+
+# Scaleway Zones and Regions
+
+Scaleway's products are deployed across multiple datacenter in the world.
+
+For performance and legal reasons, some products are splitted by Region or by Availability Zones. When using such product, you can choose the location that better fits your need (country, latency, ...).
```

Comment:
performance => technical

Reply:
I don't really see the problem with this explanation. It's in Terraform docs. I think we can suppose that there everyone who uses terraform will completely understand this doc, no?

Reply:
My bad, I misunderstood. Fixed now.